pkg/vulnsrc/osv/types.go

```diff
@@ -1,12 +1,80 @@
 package osv
 
-import "golang.org/x/vuln/osv"
+import (
+	"time"
+)
 
-// source: https://github.com/golang/vuln/blob/9d39a965865fd1d0030df18602433a01f679fd7d/osv/json.go
+type RangeType string
+
+const RangeTypeGit RangeType = "GIT"
+
+type Ecosystem string
+
+// Module identifies the Go module containing the vulnerability.
+// Note that this field is called "package" in the OSV specification.
+//
+// See https://ossf.github.io/osv-schema/#affectedpackage-field.
+type Module struct {
+	Path      string    `json:"name"`
+	Ecosystem Ecosystem `json:"ecosystem"`
+}
+
+type RangeEvent struct {
+	Introduced string `json:"introduced,omitempty"`
+	Fixed      string `json:"fixed,omitempty"`
+}
+
+type Range struct {
+	Type   RangeType    `json:"type"`
+	Events []RangeEvent `json:"events"`
+}
+
+type ReferenceType string
+
+type Reference struct {
+	Type ReferenceType `json:"type"`
+	URL  string        `json:"url"`
+}
+
+type Affected struct {
+	// The affected Go module. Required.
+	// Note that this field is called "package" in the OSV specification.
+	Module            Module            `json:"package"`
```

Comment:
They use OSV for Go vulnerabilities, so it is ok to call it Module. But we use this struct for other packages as well. I think `Package` is better.

```suggestion
	Package            Package            `json:"package"`
```

Reply:
Changed in https://github.com/aquasecurity/trivy-db/pull/326/commits/9bb432b9fbc286d7c61e1a14175b28ceae880dc7